sdk/core/System.ClientModel/tests/ModelReaderWriter/Models/AvailabilitySetDataTests.cs

```diff
@@ -17,17 +18,49 @@ internal class AvailabilitySetDataTests : ModelJsonTests<AvailabilitySetData>
 
         protected override string GetExpectedResult(string format)
         {
-            var expectedSerializedString = "{";
-            if (format == "J")
-                expectedSerializedString += "\"name\":\"testAS-3375\",\"id\":\"/subscriptions/e37510d7-33b6-4676-886f-ee75bcc01871/resourceGroups/testRG-6497/providers/Microsoft.Compute/availabilitySets/testAS-3375\",\"type\":\"Microsoft.Compute/availabilitySets\",";
-            expectedSerializedString += "\"sku\":{\"name\":\"Classic\"";
-            //if (!ignoreAdditionalProperties)
-            //    expectedSerializedString += ",\"extraSku\":\"extraSku\"";
-            expectedSerializedString += "},\"tags\":{\"key\":\"value\"},\"location\":\"eastus\",\"properties\":{\"platformUpdateDomainCount\":5,\"platformFaultDomainCount\":3}";
-            //if (!ignoreAdditionalProperties)
-            //    expectedSerializedString += ",\"extraRoot\":\"extraRoot\"";
-            expectedSerializedString += "}";
-            return expectedSerializedString; ;
+            object obj = format switch
+            {
+                "J" => new
```

Comment:
I believe using anonymous objects here are making this more clear comparing with building up literal strings